.stylelintrc.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
- [ ] Please do not create a new repo for a re-review. I noticed you started a completely new project for a re-review. Always implement suggested changes on the same branch and send the same pull request link for a re-review.

- [ ] To completely set up your linters checks, kindly copy the `.github` folder with all its contents from [this repo](https://github.com/microverseinc/linters-config/tree/master/html-css) to your repo. This will enable GitHub to run linters checks.

- [Optional] kindly edit your pull request heading. The pull request heading should be a description of what was implemented on the branch. Check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information.